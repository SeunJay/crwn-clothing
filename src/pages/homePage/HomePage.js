import React from "react";
import {HomePageContainer} from "./homePage.styles"
import "./homePage.styles.scss";
import Directory from "../../components/directory/Directory";

const HomePage = () => {
  return (
    <HomePageContainer>
      <Directory />
    </HomePageContainer>
  );
};

export default HomePage;
