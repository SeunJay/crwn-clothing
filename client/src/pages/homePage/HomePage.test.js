import React from "react";
import { shallow } from "enzyme";
import HomePage from "./HomePage"

it("should render the SignUnAndSignUp component", () => {
  const wrapper = shallow(<HomePage />);

  expect(wrapper).toMatchSnapshot();
});
