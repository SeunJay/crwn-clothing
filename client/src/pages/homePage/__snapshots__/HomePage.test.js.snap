// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the SignUnAndSignUp component 1`] = `ShallowWrapper {}`;
